CorsixTH/Lua/game_ui.lua

```diff
@@ -476,6 +476,16 @@ function GameUI:onCursorWorldPositionChange()
   if not overwindow and wx > 0 and wy > 0 and wx < self.app.map.width and wy < self.app.map.height then
     room = self.app.world:getRoom(wx, wy)
   end
+  -- Find the room associated with the current entity (Usually only applies to doors)
+  if entity and not room then
+    if entity.room then
+      room = entity.room
+    end
+    -- Special case to catch the non-dominant side of a double-door
+    if not room and entity.object_type and entity.object_type.class == "SwingDoor" then
```

Comment:
You can change this to an else if so you don't have to check `not room` again. Good thinking about the double door.

Reply:
Thanks, good spot. Comitted an update for that